grammar.js

```diff
@@ -1,169 +1,278 @@
+/**
+ * @file Yuck grammar for tree-sitter
+ * @author Philipp Mildenberger <philipp@mildenberger.me>
+ * @author Amaan Qureshi <amaanq12@gmail.com>
+ * @license MIT
+ * @see {@link https://github.com/elkowar/eww/blob/master/docs/src/configuration.md#creating-your-first-window|official syntax spec}
+ * @see {@link https://github.com/elkowar/eww| official source}
+ */
+
+// deno-lint-ignore-file ban-ts-comment
+/* eslint-disable arrow-parens */
+/* eslint-disable camelcase */
+/* eslint-disable-next-line spaced-comment */
+/// <reference types="tree-sitter-cli/dsl" />
+// @ts-check
+
+const PREC = {
+  parenthesized_expression: -1,
+  ternary: 0,
```

Comment:
I'm not sure if it's a good idea to use 0 (as this is AFAIK the default precision for everything else), I would use 1 instead, though it may be indifferent probably...

Reply:
More about being relative

Also removed parenthesized, that was unnecessary